kivy/tools/packaging/pyinstaller_hooks/rt-hook-kivy.py

```diff
@@ -29,15 +29,17 @@
 environ['KIVY_MODULES_DIR'] = join(root, 'modules')
 environ['KIVY_EMBED'] = '1'
 
+
 # Monkey-patch pygame to get around an issue with Pygame window icon and
 # PyInstaller 2.1. See kivy issue #1638
-# Uncomment the following to package pygame
-#import pygame.pkgdata
-#_original_getResource = pygame.pkgdata.getResource
-#
-#
-#def getResource(identifier, *args, **kwargs):
-#    if identifier == 'pygame_icon.tiff':
-#        raise IOError()
-#    return _original_getResource(identifier, *args, **kwargs)
-#pygame.pkgdata.getResource = getResource
+def getResource(identifier, *args, **kwargs):
```

Comment:
@matham we don't provide pygame anymore, there is not reason to look to patch pygame at runtime and this does lead to a crash for a packaged app using just SDL2.